### DISCUSSION TITLE
I need to connect to sharepoint through webdav, but it isn't working (403 Forbidden)

### DISCUSSION THREAD

**Discussion by MCorbo7** [iterate-ch/cyberduck#16018] [Q&A] [answered]
Greetings.
I need to connect to my sharepoint shared folder through webdav, I'm aware it has its own method to connect with sharepoint, but I need to connect through webdav for testing purposes.

**Comment by dkocher** [comment]
We have a pending documentation change in [^1]. Please try to set the domain in addition to the username to login using the format `DOMAIN\username` in the _Username_ input.

[^1]:  iterate-ch/docs#520

**Reply by MCorbo7** [reply]
> We have a pending documentation change in [1](#user-content-fn-1-d0acebf5c533e15ddab163e841d2ba53). Please try to set the domain in addition to the username to login using the format `DOMAIN\username` in the _Username_ input.
> 
> ## Footnotes
> 1. [Set domain to authenticate with NTLM scheme docs#520](https://github.com/iterate-ch/docs/issues/520) [↩](#user-content-fnref-1-d0acebf5c533e15ddab163e841d2ba53)

If my domain is example.sharepoint.com and my username mcorbo7@example.com then I should set the settings to
`Connection: WebDav (HTTPS)`
`URL: example.sharepoint.com`
`Username: example/mcorbo7@example.com`
`Password: **App password I had setup in my profile as I have 2FA enabled**`
`Path: /sites/admin/Shared%20documents/Forms/`

And that should work, right?

**Reply by AliveDevil** [reply, marked as answer]
In that case it is either `example\mcorbo7` (_must_ be `\`) or `mcorbo7@example.com`.

Regardless, you can use the "Microsoft Sharepoint"-protocol with OAuth authentication to connect to your Sharepoint Online without going through the hassle of WebDAV - we never tested the WebDAV connection to sharepoint.com.
